webui/react/src/ErrorHandler.ts

```diff
@@ -63,8 +62,11 @@ const handleError = (e: DaError): Error => {
   const error = e.error ? e.error : new Error(e.message);
 
   if (e.type === ErrorType.Auth) {
-    // TODO: Update to internal routing when React takes over login.
-    crossoverRoute('/ui/logout');
+    // FIXME using react router would require a react hook function or component
+    if (!window.location.pathname.endsWith('login')) {
+      const destination = `/det/login${window.location.search}`;
+      window.location.assign(destination);
+    }
```

Comment:
that worked! moving route.ts within this PR might messup the diff view of the file. I'll move and see how it looks on github otherwise sounds good : )